gpu/octree/include/pcl/gpu/octree/octree.hpp

```diff
@@ -102,12 +102,21 @@ namespace pcl
 
             /** \brief Performs search of all points within given radius on CPU. It call \a internalDownload if necessary
               * \param[in] center center of sphere
-              * \param[in] radius radious of sphere
-              * \param[out] out indeces of points within give sphere
-              * \param[in] max_nn maximum numver of results returned
+              * \param[in] radius radius of sphere
+              * \param[out] out indices of points within given sphere
+              * \param[in] max_nn maximum number of results returned
               */
             void radiusSearchHost(const PointType& center, float radius, std::vector<int>& out, int max_nn = INT_MAX);
 
+            /** \brief Performs search of all points within given radius on CPU. It call \a internalDownload if necessary
+              * \param[in] center center of sphere
+              * \param[in] radius radius of sphere
+              * \param[out] out indices of points within given sphere
+              * \param[in] max_nn maximum number of results returned
+              * \param[in] sqr_distances corresponding squared distances to query point
```

Comment:
reorder to follow the function signature. sqr_distances is a param[out]

Reply:
Don't forget this one.

Reply:
`max_nn` has a default argument, so it needs to come last. Would this be acceptable?
```cpp
            /** \brief Performs search of all points within given radius on CPU. It call \a internalDownload if necessary
              * \param[in] center center of sphere
              * \param[in] radius radius of sphere
              * \param[out] sqr_distances corresponding squared distances to query point
              * \param[out] out indices of points within given sphere
              * \param[in] max_nn maximum number of results returned
              */
            void radiusSearchHost(const PointType& center, const float radius, std::vector<float>& sqr_distances, std::vector<int>& out, const int max_nn = INT_MAX);
```

Reply:
That's not relevant so let me clarify. The current function lists parameters in the following order:
1. center
2. radius
3. out
4. sqr_distances
5. max_nn

Your docstring is using a different order, max_nn is coming before sqr_distances. The docstring should be corrected not the function signature.